content/ellipse.swift

```diff
@@ -0,0 +1,9 @@
+struct EllipseShape: Shape {
+    func path(in rect: CGRect) -> Path {
+        var path = Path()
+        let width = rect.size.width
+        let height = rect.size.height
+        path.addEllipse(in: CGRect(x: 0, y: 0, width: width, height: height))
```

Comment:
Ideally the test would also include the `x`, `y`, `width` and `height` set to some non-zero and non-full-width sizes. That way it would test if the sizes are being converted correctly, not just if the ellipse is added.